src/features/contact/contactverzoek/contactverzoekLogboek/LogboekOverzicht.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <section v-if="logboekActiviteiten?.length">
+    <utrecht-heading :level="level ? level + 1 : 3">Logboek</utrecht-heading>
+    <ul class="logboek">
+      <li
+        v-for="logboekItem in logboekActiviteiten"
+        :key="logboekItem.datum"
+        class="ita-step"
+      >
+        <utrecht-heading :level="level ? level + 2 : 4">{{
+          logboekItem.titel
+        }}</utrecht-heading>
+
+        <p>{{ logboekItem.tekst }}</p>
+        <article class="highlight" v-if="logboekItem.notitie">
+          <utrecht-heading :level="level ? level + 2 : 4"
+            >Interne toelichting</utrecht-heading
+          >
+          <p>{{ logboekItem.notitie }}</p>
+        </article>
+        <ul class="meta">
+          <li><DateTimeOrNvt :date="logboekItem.datum" /></li>
+          <li>{{ logboekItem.uitgevoerdDoor }}</li>
+          <li>
+            {{ logboekItem.kanaal ? "Kanaal: " + logboekItem.kanaal : "" }}
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </section>
+</template>
+
+<script lang="ts" setup>
+import { fetchLoggedIn, parseJson, throwIfNotOk } from "@/services";
+import { ref, watchEffect } from "vue";
+import { Heading as UtrechtHeading } from "@utrecht/component-library-vue";
+import DateTimeOrNvt from "@/components/DateTimeOrNvt.vue";
+import { fetchActor, fetchKlantcontact } from "@/services/openklant2";
+import { fetchZaakIdentificatieByUrlOrId } from "@/services/openzaak";
+
+const props = defineProps<{
+  contactverzoekId: string;
+  contactverzoekSysteemId: string;
+  level?: 1 | 2 | 3 | 4;
+}>();
+
+interface LogboekActiviteit {
+  datum: string;
+  type: string;
+  titel: string;
+  kanaal?: string | undefined;
+  tekst?: string | undefined;
+  contactGelukt?: string | undefined;
+  uitgevoerdDoor: string | undefined;
+  notitie?: string | undefined;
+}
+
+const logboekActiviteiten = ref<LogboekActiviteit[]>([]);
+
+const activiteitTypes = {
+  klantcontact: "klantcontact",
+  toegewezen: "toegewezen",
+  verwerkt: "verwerkt",
+  zaakGekoppeld: "zaak-gekoppeld",
+  zaakkoppelingGewijzigd: "zaakkoppeling-gewijzigd",
+  interneNotitie: "interne-notitie",
+};
+
+watchEffect(async () => {
+  logboekActiviteiten.value = [];
+  await fetchLoggedIn(
+    `/api/logboek/api/v2/objects?data_attr=heeftBetrekkingOp__objectId__exact__${props.contactverzoekId}`,
+  )
+    .then(throwIfNotOk)
+    .then(parseJson)
+    .then(
+      async (r) =>
+        (logboekActiviteiten.value = await mapAndEnrichLogboek(r.results)),
+    );
+});
+
+const mapAndEnrichLogboek = async (
+  logboek: any,
```

Comment:
liever geen any. kunnen we een type definitie maken voor het logboek zoals we dat van de backend verwachten terug te krijgen?